### PR TITLE
add e2e set-up instructions

### PR DESCRIPTION
**Pull request by satvu**
### Issue describing the changes in this PR

resolves #issue_for_this_pr

### Pull request checklist

* [x] My changes **do not** require documentation changes
  * [ ] Otherwise: Documentation issue linked to PR
* [x] My changes **should not** be added to the release notes for the next release
  * [ ] Otherwise: I've added my notes to `release_notes.md`
* [x] My changes **do not** need to be backported to a previous version
  * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [x] I have added all required tests (Unit tests, E2E tests)

### Additional information

Add the requirements and instructions for running E2E tests locally.

### CODE REVIEW

**Review by kshyju — APPROVED** [review]
Thank you! Going to try these instructions myself.

File: README.md [hunk 1]
```diff
@@ -88,6 +88,21 @@ Under the hood, Rider launches the Core Tools with the `--dotnet-isolated-debug`
 
 You can place a breakpoint in any function, and inspect your code as it is running. Note that [debugging startup code may timeout (#434)](https://github.com/Azure/azure-functions-dotnet-worker/issues/434).
 
+## Running E2E Tests
+
+### Requirements
+
+- [Powershell 7](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-windows?view=powershell-7.2)
+- [CosmosDb Emulator](https://docs.microsoft.com/en-us/azure/cosmos-db/local-emulator?tabs=ssl-netstd21)
+- [Azure Storage Emulator](https://docs.microsoft.com/en-us/azure/storage/common/storage-use-emulator)
```
Comment (fabiocav) [inline README.md:97]: Is this a valid requirement? The setup should install Azurite, correct?

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (liliankasem) [inline .vscode/settings.json:1]: I don't think we should check this in

**Review by liliankasem — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (liliankasem) [reply README.md]: I think it's always worth noting all the "things" that will be used / are needed for the E2E tests to run. You can add a note that the script will download this for you 🤷‍♀️

Comment (fabiocav) [reply README.md]: My concern is that listing those as requirements at the beginning, with no additional information, will likely push people following this to go install those components before proceeding, even though this is handled by the script. It's fine to mention what the script will do, but I wouldn't position this as a requirement/pre-requisite to move forward with the process, which is what this implies.

**Pull request merged**